apps/mobile-wallet/src/features/receive/screens/QRCodeScreen.tsx

```diff
@@ -51,39 +28,9 @@ const QRCodeScreen = ({ navigation, route: { params }, ...props }: ScreenProps)
       )}
       {...props}
     >
-      <ScreenSection centered>
-        <QRCodeContainer>
-          <QRCode size={200} value={params.addressHash} />
-        </QRCodeContainer>
-      </ScreenSection>
-      <ScreenSection centered verticalGap>
-        <Button short title={t('Copy address')} onPress={handleCopyAddressPress} iconProps={{ name: 'copy' }} />
-        <Button short title={t('Buy')} onPress={openBuyModal} iconProps={{ name: 'credit-card' }} />
```

Comment:
I see that you removed the "Buy" button. In my opinion, it was a good idea of Diomark's to add it there.

Reply:
It really bothered me tbh.

1) The 2 buttons UI was pretty bad, moving content off screen in most phone screen sizes.
2) The buy button is literally accessible just next to the receive button in the dashboard and the addresses screen. I see no value to duplicate actions that are so close.
3) I found no other wallets doing this.